README.md

```diff
@@ -1,4 +1,52 @@
-> [!IMPORTANT]  
-> Under Construction.
+# bazelrc Presets
+
+Bazel has a tremendously large number of flags.
+Many are obscure, many are important to use, and many have an undesirable default value.
+
+Bazel options may be stored in `*.bazelrc` files, in several places on disk.
+Read [the Bazel bazelrc documentation](https://bazel.build/run/bazelrc).
+This repo provides bazelrc files you may vendor into your repo, which we call "presets".
+
+> [!NOTE]  
+> Ideally bazelrc would allow import statements from external repositories, so that you wouldn't be forced to copy presets into your repo.
+> However the current behavior is that imports must be repository-relative.
+> This could also be considered a feature, since it means that newer preset changes will be code-reviewed when they arrive in your repo,
+> rather than as an invisible side-effect of updating the version of bazelrc-presets.
```

Comment:
Probably opinionated, but I try to stick to 1 sentence per line for a few reasons:

- you don't have to mess around with the line length if you extend a sentence that is split over multiple lines
- usually editors have soft wrap so a user might not notice

Reply:
Yes that's been my preference as well, this was just some copy-pasta from previous repo...